pkg/metrics/collector/pod_ip_metrics.go

```diff
@@ -252,19 +245,20 @@ func (c *podIpMetricsCollector) updateReuseIpStats(e fsnotify.Event, f string, b
 		if ok {
 			oldT := reuseIp.ipReleasedTimestamp
 			diff := uint64(c.clock.Now().Sub(oldT).Milliseconds())
-			reuseIp.ipReuseInterval = float64(diff)
-
-			c.reuseIps.count += 1
-			c.reuseIps.sum += float64(diff)
-			if diff < c.reuseIps.min {
-				c.reuseIps.min = diff
+			if diff > 0 {
```

Comment:
Do we need this check? Will the 'diff' ever be less than zero?

Reply:
I agree, theoretically it won't be less than zero, but it is possible to be 0, I think it would be safer to add this check.